site/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="author" content="shreekishna-jpg">
+  <meta name="description" content="A brief description of the page">
+  <title>Title</title>
+  <link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96" />
+  <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+  <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"/>
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+  <link rel="manifest" href="/site.webmanifest" />
+</head>
+
+<body>
+  <div class="width-wrap">
+  <header id="top">
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="glossary.html">Glossary</a></li>
+        <li><a href="links.html">Links</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section id="Thumbnails-and-Alternative-Sizes">
+      <a href="images/myself.png">
+        <picture>
+          <source media="(max-width: 500px)" srcset="images/thumbnail_150px.png">
+          <source media="(min-width: 501px)" srcset="images/thumbnail_300px.png">
+          <img src="images/thumbnail_300px.png" alt="Thumbnail-of-Myself">
+        </picture>
+      </a>
+    </section>
```

Comment:
> 1. `Lab 3 > Thumbnails and Alternative Sizes > img uses sizes attribute`
> 2. `Lab 3 > Thumbnails and Alternative Sizes > img srcset is setup correctly`
> 3. `Lab 3 > Thumbnails and Alternative Sizes > displays the 150px thumbnail when the viewport is under 500px`
> 4. `Lab 3 > Thumbnails and Alternative Sizes > displays the 300px thumbnail when the viewport is over 500px`
> 5. `Lab 3 > Thumbnails and Alternative Sizes > click on the displayed thumbnail opens the full image`

Was expected to be nested under the `bio` section.
```suggestion
    <section id="Thumbnails-and-Alternative-Sizes">
      <a href="images/myself.png" target="_blank">
        <picture>
          <img src="images/thumbnail_300px.png" alt="Thumbnail-of-Myself" sizes="(max-width:150px) 150px, 300px" srcset="images/thumbnail_150px.png 150x, images/thumbnail_300px.png 300w">
        </picture>
      </a>
    </section>
```

> 6. `Lab 3 > Thumbnails and Alternative Sizes > alt attribute is supplied to the images`

Overridden, alt not found due to nesting issues.